data/hai missions.txt

```diff
@@ -435,3 +435,585 @@ mission "Unfettered returning home"
+	minor
+	source
+		government "Hai"
+	destination "Vinci"
+	passengers 2
+	to offer
+		random < 30
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`While eating in a spaceport cafeteria that serves both Hai and human cuisine, you are approached by a teenage human and Hai. "Hello. Do you know how to get to <destination>?" the Hai asks.`
+			choice
+				`	"I do. Why do you want to go there?"`
+				`	"I don't. You'll have to ask someone else."`
+					decline
+			`	The human responds, "There's a massive convention on <planet> for a game called 'Boundless Frontiers.' The convention has a huge cosplaying scene, and I think Kiru would fit right in."`
+			`	"Arthur and I have <payment> to give you if you bring us to the <planet>," Kiru says. "We can find our own way back to the Hai space, as well."`
+			choice
+				`	"Alright, I can take you there."`
+					accept
+				`	"Sorry, you'll have to find someone else to bring you."`
+					decline
+					
+	on complete
+		payment 75000
+		dialog `Arthur and Kiru pay you <payment> before running off to the convention center. As they run away, you notice that Kiru is drawing a number of eyes from the crowd, but no one seems to suspect that he is an actual alien.`
+
+
+
+mission "Hai Honeymoon"
+	description "Bring Anaya and Touhar to <destination> for their honeymoon."
+	minor
+	source "Skyfarm"
+	destination "Featherweight"
+	passengers 2
+	to offer
+		random < 60
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`You hop around the spaceport of <origin>, trying to adjusted to the low gravity of this moon, when you are approached by a toweringly tall human woman and an equally tall Hai male who are clearly having no trouble with <origin>'s gravity.`
+			`	"You look like you need help there, Captain," the woman says with a chuckle.`
+			choice
+				`	"Don't worry. I almost have the hang of it."`
+				`	"Maybe a little bit."`
+			`	"It's always fun watching captains when they step off the comfort of their ship," the Hai says.`
+			`	"At least he's trying, Touhar," the woman says.`
+			choice
+				`	"Can I help you two with something?"`
+				`	"Is there something I can help you with, or are you just going to watch me?"`
+			`	"Actually, there is something," the woman says. "My name is Anaya, and this is my husband, Touhar. And you are?"`
+			choice
+				`	"My name is <first> <last>."`
+				`	"Wait, your husband?"`
+					goto husband
+			
+			`	"Very nice to meet you, Captain <last>," Touhar says to you.`
+				goto next
+			
+			label husband
+			`	"What, you've never seen an interspecies couple before?" Touhar asks with a hint of sarcasm in his voice.`
+			
+			label next
+			`	"We got married last week, and we were hoping to travel to <destination> for our honeymoon. I know it's risky to bring Touhar into human space, but <planet> is a sparsely inhabited world, and I have a friend who owns a house in the mountains far from anyone who could see Touhar."`
```

Comment:
"Touhar" is used wrice in the same sentence. One of those occurrences should probably be replaced with "him" (most likely the second one).